src/main/java/com/iopipe/plugin/profiler/ProfilerExecution.java

```diff
@@ -176,6 +204,7 @@ final void __post()
 				// compressed anyway. The header and footer could be compressed
 				// but that would probably save only a dozen bytes so the
 				// loss of speed compressing compressed data is pointless
+				// Just put no effort into it.
```

Comment:
Is this extra comment necessary to add?

Reply:
I will undo that, not too sure why I put that in there now.